vllm/model_executor/model_loader/loader.py

```diff
@@ -702,41 +725,16 @@ def save_model(
         max_size: Optional[int] = None,
     ) -> None:
         from vllm.distributed import get_tensor_model_parallel_rank
-        from serverless_llm_store._C import save_tensors
+        from serverless_llm_store import save_dict
         
-        # if pattern is None:
-        #     pattern = ShardedStateLoader.DEFAULT_PATTERN
         rank = get_tensor_model_parallel_rank()
         state_dict = ServerlessLLMLoader._filter_subtensors(model.state_dict())
         
         # move all tensors to CPU
         for key, tensor in state_dict.items():
             state_dict[key] = tensor.cpu()
-        
-        tensor_names = list(state_dict.keys())
-        tensor_data_index = {}
-        for name, param in state_dict.items():
-            param_storage = param.untyped_storage()
-            data_ptr = param_storage.data_ptr()
-            size = param_storage.size()
-            tensor_data_index[name] = (data_ptr, size)
-        
-        print(tensor_data_index)
-        rank_path = path + f"rank_{rank}" 
-        if not os.path.exists(rank_path):
-            os.makedirs(rank_path)
-        # save tensors
-        tensor_offsets = save_tensors(tensor_names, tensor_data_index, rank_path)
-        
-        # create tensor index
-        tensor_index = {}
-        for name, param in state_dict.items():
-            # name: offset, size
-            tensor_index[name] = (tensor_offsets[name], tensor_data_index[name][1], tuple(param.shape), tuple(param.stride()), str(param.dtype))
-
-        # save tensor index
-        with open(os.path.join(rank_path, "tensor_index.json"), "w") as f:
-            json.dump(tensor_index, f)
+            
+        save_dict(state_dict, os.path.join(path, f"rank_{rank}"))
```

Comment:
I didn't found the save dict function under serverless_llm_store, where can I build the latest version with `save_dict` function?

Reply:
use latest xly/fix-docker-build from serverlessllm